apis/server/network_bridge.go

```diff
@@ -43,9 +43,9 @@ func (s *Server) listNetwork(ctx context.Context, rw http.ResponseWriter, req *h
 		return err
 	}
 
-	respNetworks := types.NetworkListResp{Networks: []*types.NetworkInfo{}, Warnings: nil}
+	respNetworks := []types.NetworkResource{}
 	for _, net := range networks {
-		respNetworks.Networks = append(respNetworks.Networks, &types.NetworkInfo{
+		respNetworks = append(respNetworks, types.NetworkResource{
```

Comment:
There are only three fields we uses. 

In the future, we need to list which fields we need and make it perfect.

For now, LGTM